addons/website_slides/static/src/js/snippets.animation.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import publicWidget from '@web/legacy/js/public/public_widget';
+
+publicWidget.registry.WebsiteAnimate.include({
```

Comment:
Detail but, technically, as you include something that is defined in website, it would be better to require that file in the `import` too. Otherwise, if for some reason this file here comes after website with some refactoring, it won't work anymore (even though JS file order should be irrelevant with the JS system module).